### DISCUSSION TITLE
Anyone used bodmer/TFT_eWidget library with CYD and got touch working?

### DISCUSSION THREAD

**Discussion by peteDDD** [witnessmenow/ESP32-Cheap-Yellow-Display#96] [General]
Anyone used bodmer/TFT_eWidget library with CYD and got touch working?

**Comment by dpwlt** [comment]
I managed to get it working using TFT_eSPI for the display and TFT-touch for the touch panel. You'll need to use the modified user_setup.h for the CYD. This is on the version with the USB-C port.

I've uploaded a sketch I used for figuring it out here: https://github.com/dpwlt/public/blob/main/CYD/CYD_touch/CYD_touch.ino
